src/turing_inference.jl

```diff
@@ -8,7 +10,10 @@ function turing_inference(
     likelihood = (u,p,t,σ) -> MvNormal(u, σ[1]*ones(length(u))),
     num_samples=1000, sampler = Turing.NUTS(0.65),
     syms = [Turing.@varname(theta[i]) for i in 1:length(priors)],
-    progress = false, kwargs...,
+    obsvbls = 1:size(data, 1),
```

Comment:
it would be better to just use `saveidxs` here, since that would reduce the memory use.

Reply:
Will do that.